cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1478,6 +1484,11 @@ private List<VolumeMount> getVolumeMounts() {
             }
         }
 
+        if (authorization instanceof KafkaAuthorizationOpa) {
+            KafkaAuthorizationOpa opaAuthz = (KafkaAuthorizationOpa) authorization;
+            volumeMountList.addAll(AuthenticationUtils.configureOauthCertificateVolumeMounts("authz-opa", opaAuthz.getTlsTrustedCertificates(), OAUTH_TRUSTED_CERTS_BASE_VOLUME_MOUNT + "/authz-opa-certs"));
```

Comment:
Maybe you can rename the `OAUTH_TRUSTED_CERTS_BASE_VOLUME_MOUNT` variable to something more generic as it is not used anymore just for OAuth? E.g. `TRUSTED_CERTS_BASE_VOLUME_MOUNT`?